Dockerfile

```diff
@@ -28,7 +31,7 @@ RUN adduser -D $USER -u 1000 \
 
 USER 1000
 WORKDIR /
-COPY --from=BACK --chown=$USER:$USER /go/src/casdoor/server ./server
+COPY --from=BACK --chown=$USER:$USER /go/src/casdoor/server_${BUILDX_ARCH} ./server
```

Comment:
Why change this folder?

Reply:
This change was already present in the previous implementation. See https://github.com/casdoor/casdoor/commit/9aec69ef4744f5f49ef8a6605a44a17eada8c65d#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L34

The Go build process builds two versions for AMD64 and ARM64. That’s the reason for the line change. The change itself does not change any folder. It’s the name of the binary from the build step that is different. See also https://github.com/casdoor/casdoor/commit/9aec69ef4744f5f49ef8a6605a44a17eada8c65d#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823L11

Reply:
@AlexanderBabel will this cause any difference in the final amd64 Docker image?

Reply:
No, the filename of the binary in the final images is unaffected by the change.

Reply:
You can see that, because the second part of the copy command is still `./server` that is the name of the binary in the final images.